receiver/libhoneyreceiver/internal/simplespan/simplespan.go

```diff
@@ -0,0 +1,187 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package simplespan // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/libhoneyreceiver/internal/simplespan"
```

Comment:
I'd prefer we don't use the term `span` since this could represent a span or a log.  Maybe `event`?

Reply:
I'm happy to use event if it's wouldn't imply a special log https://opentelemetry.io/docs/specs/otel/logs/event-api/ 

Would something like "lhevent" work since it's specific to the libhoney object shape?

Reply:
Good point about otel events. Something honeycomb-specific like libhoneyEvent is good.